src/annotator/util/extract-annotation-query.js

```diff
@@ -1,41 +1,47 @@
 'use strict';
 
 /**
- * Return the `#annotations:*` ID or query from the given URL's fragment.
+ * Return the `#annotations:*` ID from the given URL's fragment.
  *
- * If the URL contains a `#annotations:query:*` (or `#annotatons:q:*`) fragment
- * then return a `{query: *}` object containing the query extracted from the
- * fragment.
+ * If the URL contains a `#annotations:<ANNOTATION_ID>` fragment then return
+ * the annotation ID extracted from the fragment. Otherwise return `null`.
  *
- * If the URL contains a `#annotations:<ANNOTATION_ID>` fragment then return a
- * `{annotations: *}` object containing the annotation ID extracted from the
- * fragment.
+ * @param {string} url - The URL to extract the ID from.
+ * @return {string|null} - The extracted ID, or null.
+ */
+function annotations(url) {
+  // Annotation IDs are url-safe-base64 identifiers
+  // See https://tools.ietf.org/html/rfc4648#page-7
+  var annotFragmentMatch = url.match(/#annotations:([A-Za-z0-9_-]+)$/);
+  if (annotFragmentMatch) {
+    return annotFragmentMatch[1];
+  }
+  return null;
+}
```

Comment:
Here's what the code for reading this setting  used to be (before this and a previous PR) :)

```
function extractAnnotationQuery(url) {
  var filter = {};
  try {
    // Annotation IDs are url-safe-base64 identifiers
    // See https://tools.ietf.org/html/rfc4648#page-7
    var annotFragmentMatch = url.match(/#annotations:([A-Za-z0-9_-]+)$/);
    var queryFragmentMatch = url.match(/#annotations:(query|q):(.+)$/i);
    if (queryFragmentMatch) {
      filter.query = decodeURI(queryFragmentMatch[2]);
    } else if (annotFragmentMatch) {
      filter.annotations = annotFragmentMatch[1];
    } else {
      filter = null;
    }
  } catch (err) {
    // URI Error should return the page unfiltered.
    filter = null;
  }
  return filter;
}
```